frontend/src/modules/modals/MultiStepWizard.tsx

```diff
@@ -529,9 +473,11 @@ const MultiStepWizard = (): JSX.Element => {
    */
   const doSetAlarmValues = (min: number, max: number) => {
```

Comment:
Can we rename this function? I assume `setAlarmValues` was already taken, but maybe we can find a clearer name.